### PR TITLE
bump version number for 0.8.0

### PR DESCRIPTION
**Pull request by audrasjb**
Let's not forget to bump version number for version 0.8.0 🤡

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ Contributors: wordpressdotorg, audrasjb, whodunitagency, pbiron, whyisjake, xkon
 Requires at least: 5.3
 Tested up to: 5.4
 Requires PHP: 5.6
-Stable tag: 0.7.0
+Stable tag: 0.8.0
```
Comment (pbiron) [inline readme.txt:6]: Need to also update the version number in `package.json` :-)

**Review by pbiron — CHANGES_REQUESTED** [review]

File: readme.txt (hunk 1 above)
Comment (audrasjb) [reply readme.txt:6]: and package-lock.json

Comment (audrasjb) [reply readme.txt:6]: on it!

Comment (audrasjb) [reply readme.txt:6]: updated

**Review by pbiron — APPROVED** [review]

**Pull request merged**